src/NuGet.CommandLine/MsBuildUtility.cs

```diff
@@ -68,7 +68,7 @@ public static IEnumerable<string> GetProjectReferences(string msbuildPath, strin
                 "/p:BuildProjectReferences=false " +
                 $"/p:CustomAfterMicrosoftCommonTargets={targetPath} " +
                 $"/p:ResultsFile={resultsPath} " +
```

Comment:
Maybe all wrap the other paths in quotes (in case the temp dir is mapped to one with spaces)?